pymc3/variational/approximations.py

```diff
@@ -382,56 +323,21 @@ def cov(self):
         x = (self.histogram - self.mean)
         return x.T.dot(x) / pm.floatX(self.histogram.shape[0])
 
-    @classmethod
-    def from_noise(cls, size, jitter=.01, local_rv=None,
-                   start=None, model=None, random_seed=None, **kwargs):
-        """Initialize Histogram with random noise
-
-        Parameters
-        ----------
-        size : `int`
-            number of initial particles
-        jitter : `float`
-            initial sd
-        local_rv : `dict`
-            mapping {model_variable -> local_variable}
-            Local Vars are used for Autoencoding Variational Bayes
-            See (AEVB; Kingma and Welling, 2014) for details
-        start : `Point`
-            initial point
-        model : :class:`pymc3.Model`
-            PyMC3 model for inference
-        random_seed : None or `int`
-            leave None to use package global RandomStream or other
-            valid value to create instance specific one
-        kwargs : other kwargs passed to init
-
-        Returns
-        -------
-        :class:`Empirical`
-        """
-        hist = cls(
-            None,
-            local_rv=local_rv,
-            model=model,
-            random_seed=random_seed,
-            **kwargs)
-        if start is None:
-            start = hist.model.test_point
+    @node_property
+    def std(self):
+        return tt.sqrt(tt.diag(self.cov))
+
+    def __str__(self):
+        if isinstance(self.histogram, theano.compile.SharedVariable):
+            shp = ', '.join(map(str, self.histogram.shape.eval()))
         else:
-            start_ = hist.model.test_point.copy()
-            update_start_vals(start_, start, hist.model)
-            start = start_
-        start = pm.floatX(hist.gbij.map(start))
-        # Initialize particles
-        x0 = np.tile(start, (size, 1))
-        x0 += pm.floatX(np.random.normal(0, jitter, x0.shape))
-        hist.histogram.set_value(x0)
-        return hist
+            shp = 'None, ' + str(self.ddim)
+        return '{cls}[{shp}]'.format(shp=shp, cls=self.__class__.__name__)
 
 
-class NormalizingFlow(Approximation):
-    R"""
+class NormalizingFlowGroup(Group):
+    R"""Normalizing Flow Group
+
```

Comment:
If p(z_0) is the standard normal distribution in this group, it should be explicitly explained.